pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md

```diff
@@ -51,5 +57,59 @@ Calico is now pre-installed in a DC/OS cluster and can be used by containers to
 Metronome based jobs can now join container networks to communicate with other services/jobs in the same network.
 
 ## Domain Sockets for Agent Executor Communication
-
 Agents and Executors now communicate over Unix Domain sockets making operators life easy in the presence of container overlay networks.
+### Deprecation of Marathon LB
+
+### Hive Metastore for Spark SQL
+
+### Add support for 3rd party CSI Driver in DSS
+
+### Secure Kafka TLS with User Provided TLS Certs (HPE GIT)
+
+### Certificate ( HPE GIT)
+
+### 3rd Party DVDI driver usage (HPE GIT)
+
+### Jenkins Support for DC/OS Windows
+
+### Mesos Operator API scalability
+
+### Advanced Jobs w/ Dependencies
+
+### Configurable Mesos Flags to enable easy install time config
+
+### Fault tolerance for Exhibitor to enable failover
+
+## Issues fixed in this release
+
+The issues that have been fixed in DC/OS 2.1.0 are listed below. Most change descriptions include one or more issue tracking identifiers enclosed in parenthesis for reference
+
+
+COPS-5951
+COPS-5915
+COPS-5868
+COPS-5827
+COPS-5814
+COPS-5804
+COPS-5738
+COPS-5629
+COPS-5617
+COPS-5615
+COPS-5428
+### Marathon Fixed and Improved Issues
+
+https://github.com/mesosphere/marathon/blob/master/changelog.md
+
+## Breaking changes
+Remove the octarine package from DC/OS. It was originally used as a proxy for the CLI but is not used for this purpose, anymore.
+
+DC/OS Net: wait till agents become active before fanning out Mesos tasks. (DCOS_OSS-5463)
```

Comment:
"Wait". 

@jkoelker  Is there a better way to describe this?

Reply:
```suggestion
DC/OS Net will now wait until agents become active before adding DNS entries for tasks on the agent to prevent resolving to unreachable addresses (DCOS_OSS-5463)
```